errors.go

```diff
@@ -197,7 +197,11 @@ func checkForErrors(res *http.Response) error {
 	case 403:
 		return ErrForbidden
 	case 404:
-		return ErrNotFound
+		if string(out) != "" {
+			return errors.New(string(out))
```

Comment:
Would it maybe be better to have an error type with a string variable for the message. That way SDK consumers could still check that the error was a ErrAppNotFound, but still be able to access the error message as well?